tests/metrics/test_MISim.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from sdgx.metrics.pair_column.mi_sim import MISim
+
+# 创建测试数据
+real_data_discrete = pd.DataFrame(
+    {
+        "col1": ["a", "b", "c", "d", "e"],
+        "col2": ["a", "b", "b", "b", "e"],
+    }
+)
```

Comment:
please change these test dataset in  `pytest.fixture`(as we discussed before)

```suggestion
@pytest.fixture
def real_data_discrete():
    yield pd.DataFrame(
       {
           "col1": ["a", "b", "c", "d", "e"],
           "col2": ["a", "b", "b", "b", "e"],
       }
   )
```